Source/WebKit/UIProcess/Launcher/cocoa/ProcessLauncherCocoa.mm

```diff
@@ -81,19 +81,18 @@
 
 static bool shouldLeakBoost(const ProcessLauncher::LaunchOptions& launchOptions)
 {
-#if PLATFORM(IOS_FAMILY)
-    UNUSED_PARAM(launchOptions);
-    // On iOS, we don't need to leak a boost message because RunningBoard process assertions give us the
-    // right priorities.
-    return false;
-#else
     // On Mac, leak a boost onto the NetworkProcess and GPUProcess.
+#if PLATFORM(MAC)
 #if ENABLE(GPU_PROCESS)
     if (launchOptions.processType == ProcessLauncher::ProcessType::GPU)
         return true;
 #endif
-    return launchOptions.processType == ProcessLauncher::ProcessType::Network;
-#endif
+    if (launchOptions.processType == ProcessLauncher::ProcessType::Network)
+        return true;
```

Comment:
This was the cause of the previous page load regression. This boost prevents the network and GPU processes from getting throttled by App Nap unless the UI process is napping. So we still need the boost to match performance behavior.